src/components/VaccineProgressBar/VaccineProgressBar.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { ParentSize } from '@vx/responsive';
+import { v4 as uuidv4 } from 'uuid';
+import { ProgressBarContainer, StyledSvg } from './VaccineProgressBar.style';
+import { COLOR_MAP, VACCINATIONS_COLOR_MAP, vaccineColor } from 'common/colors';
+import { formatPercent } from 'common/utils';
+
+export interface ProgressBarProps {
+  // TODO(michael): Remove once we migrate the location page to the new version.
+  oldVersion?: boolean;
+  locationName: string;
+  vaccinationsInitiated: number;
+  vaccinationsCompleted: number;
+}
+
+function getOffsetPercentage(decimal: number) {
+  return formatPercent(decimal, 1);
+}
+
+const VaccineProgressBar: React.FC<ProgressBarProps & { width: number }> = ({
+  oldVersion = false,
+  vaccinationsInitiated,
+  vaccinationsCompleted,
+  locationName,
+  width,
+}) => {
+  const height = 16;
+  const color = oldVersion
+    ? VACCINATIONS_COLOR_MAP.COMPLETED
+    : vaccineColor(vaccinationsInitiated);
+
+  const titleId = uuidv4();
+  const hatchPatternId = uuidv4();
+
+  return (
+    <StyledSvg
```

Comment:
I ended up implementing the rounded corners as CSS styles on the `<svg>` element... I was a bit surprised it worked, but seems to. 😁

Reply:
love it! also- the clipPath approach that you mentioned in slack was on point. i referenced some code of @pnavarrc 's when making [the vaccinations thermometer](https://github.com/covid-projections/covid-projections/pull/3862/files#diff-aabffd570f9c3dba1325edad05098d39ce775fdc3876f20a273a0bdcc01bf4dcR51), which uses clipPath to round the therm's edges. now we know!💡